tests/test_managers_recurrence.py

```diff
@@ -1,14 +1,11 @@
 from datetime import datetime
+
 from django.utils.timezone import make_aware
 from recurrence import choices
 from recurrence.models import Date, Recurrence, Rule
 import pytest
-import pytz
-import recurrence
 
-
-def aware(dt):
-    return make_aware(dt, pytz.utc)
```

Comment:
Probably best to use the stock `make_aware` and without timezone definitions if feasible? The timezone should be overridden with the Django `TIME_ZONE` flag in the test settings if timezone specializations are desired.